platform/fabric/driver/config.go

```diff
@@ -9,6 +9,7 @@ package driver
 import (
 	"time"
 
+	"github.com/hyperledger-labs/fabric-smart-client/platform/common/core/generic/committer"
```

Comment:
the driver should not have dependency to the core. It should be the other way around